src/meta/processors/admin/HBProcessor.cpp

```diff
@@ -29,6 +29,17 @@ void HBProcessor::process(const cpp2::HBReq& req) {
         return;
     }
 
+    ClusterID peerCluserId = req.get_clusterId();
+    if (peerCluserId == 0) {
```

Comment:
Maybe you need to add some ut to test when peerCluserId = 0 or peerCluserId not match.

Reply:
added!